### ISSUE TITLE
Images without a ground truth mask or bounding box

### ISSUE THREAD

**Issue by rain-m** [matterport/Mask_RCNN#330]
Hi, 
I would like to add to my training set negative images, which do not contain any object of interest (No ground truth mask or bbox). 
I managed to start training the network with this additional data. However, to my understanding of the implementation of the loss function, these images do not contribute to the loss. 
Does anyone know, if it is possible to add images to the training, which do not contain any ground truth bounding box?

mrcnn_bbox_loss_graph: https://github.com/matterport/Mask_RCNN/blob/master/model.py#L1092
rpn_bbox_loss_graph: https://github.com/matterport/Mask_RCNN/blob/master/model.py#L1026
Thanks
